src/bmffimage.cpp

```diff
@@ -452,6 +506,32 @@ uint64_t BmffImage::boxHandler(std::ostream& out /* = std::cout*/, Exiv2::PrintS
     case TAG_xml:
       parseXmp(buffer_size, io_->tell());
       break;
+    case TAG_brob: {
+      enforce(data.size() >= 4, Exiv2::ErrorCode::kerCorruptedMetadata);
+      uint32_t realType = data.read_uint32(0, endian_);
+      if (bTrace) {
+        out << "type: " << toAscii(realType);
+      }
+#ifdef EXV_HAVE_BROTLI
+      DataBuf arr;
+      brotliUncompress(data.c_data(4), data.size() - 4, arr);
+      if (realType == TAG_exif) {
+        uint32_t offset = arr.read_uint32(0, endian_) + 4;
+        if (arr.size() > 8) {
```

Comment:
```suggestion
        enforce(offset <= arr.size(), Exiv2::ErrorCode::kerCorruptedMetadata);
        if (arr.size() > 8) {
```

Reply:
Nice catch, thanks. I rolled in the min header length in there and got rid of the if clause while at it.